docs/webserver.md

```diff
@@ -0,0 +1,41 @@
+# Webserver
+
+| | web02.rackspace.theforeman.org |
+| - | - |
+| type | OpenStack VM |
+| OS | CentOS 7 |
+| CPUs | 2 |
+| RAM | 2GB |
+| Storage | /dev/xvda (40GB): root, /dev/xvdb (100GB) + /dev/xvdc (50GB): data |
+| Managed by | [web.pp](https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/profiles/manifests/web.pp) |
+
+## Domains
+
+These domains are all hosted on the webserver.
+
+* theforeman.org, www.theforeman.org
+* deb.theforeman.org
+* debugs.theforeman.org
+* downloads.theforeman.org
+* stagingdeb.theforeman.org
+* yum.theforeman.org
+* rsync.theforeman.org
+
+## Volumes
+
+/var/www is mounted on a separate 140GB block device via LVM.  /var/www/freight* contain the staging areas for freight (deb), and /var/www/vhosts contain the web roots themselves.
+
+## Firewall
+
+firewalld is manually configured (non-puppetized) with TCP ports:
```

Comment:
There is no firewall *on* the new host (but OpenStack has one)

Reply:
Do you want this to reflect the new webserver before being merged or merge as is for the old one and do a round of updates?

Reply:
I think workflow wise it's better to change that when the new machine becomes prod, as that's what we're documenting here after all.